src/main/webapp/resources/swagger-ui/carecode-channceling-apis.yaml

```
@@ -0,0 +1,2146 @@
+  description: View a Booking details in the System
+- name: Reverse a Booking
+  description: Reverse a Booking in the System
+
+
+
+paths:
+  /specializations:
+    
+    post:
+      security:
+        - ApiKeyAuth: []
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)